charts/aws-fluent-bit/values.yaml

```diff
@@ -1,18 +1,26 @@
 namespace:
-  create: true
+## Namespace creation is set to false as the namespace is expected to be created through the CLI since eksctl command may require.
+  create: false
   name: "calyptia-fluentbit"
 
+## The service account creation configuration is taken care by the eksctl command to contact the AWS marketplace API's to fetch the image.
 serviceAccount:
-  name: "eks-calyptia-fluentbit"
-  create: true
+  create: false
+## This is the name of the service account created by the eksctl create iamservice command.
+  name: "<Enter the name of the service account>"
```

Comment:
I would say let us default it and use that default name in the commands to create it - simplifies things for people to copy-paste. We can of course say "you can change the name but also update the values file".